internal/service/flexcluster/resource_schema.go

```diff
@@ -145,20 +146,52 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 				},
 				MarkdownDescription: "Method by which the cluster maintains the MongoDB versions.",
 			},
+			"delete_on_create_timeout": schema.BoolAttribute{
+				Optional: true,
+				PlanModifiers: []planmodifier.Bool{
+					customplanmodifier.CreateOnlyBoolPlanModifier(),
+				},
+
+				MarkdownDescription: "Indicates whether to delete the resource if creation times out. Default is `true`. When Terraform apply fails, it returns immediately without waiting for cleanup to complete. If you suspect a transient error, wait before retrying to allow resource deletion to finish.",
```

Comment:
```suggestion
				MarkdownDescription: "Indicates whether to delete the created resource if a timeout is reached when waiting for completion. Default is `true`. If you suspect a transient error, wait before retrying to allow resource deletion to finish.",
```
The 3rd sentence was confusing to me, would just leave as one description.

Reply:
This is following the description in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/3515/files#diff-e9d48cdf6bde5c047a1a852d6c3275175fe0a3d47f7ab543facdf8663b57fc57R334 which was reviewed by docs team.
I am open to changing it, but I would loop in docs team to review this and then change it in all resources.